testing/assimp/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Russ Webber <russ@rw.id.au>
+# Maintainer: Russ Webber <russ@rw.id.au>
+pkgname=assimp
+pkgver=4.1.0
+pkgrel=0
+pkgdesc="Open Asset Import Library imports and exports 3D model formats."
+url="http://www.assimp.org/"
+arch="all"
+license="BSD-3-Clause"
+makedepends="cmake zlib-dev minizip-dev"
+subpackages="$pkgname-dev"
+source="assimp-$pkgver.tar.gz::https://github.com/assimp/assimp/archive/v$pkgver.tar.gz
+        01-rm-revision-test.patch"
+builddir="$srcdir/assimp-$pkgver"
+
+build() {
+  if [ "$CBUILD" != "$CHOST" ]; then
+  CMAKE_CROSSOPTS="-DCMAKE_SYSTEM_NAME=Linux -DCMAKE_HOST_SYSTEM_NAME=Linux"
+  fi
+  cmake \
+  -DCMAKE_INSTALL_PREFIX=/usr \
```

Comment:
need indent from here

Reply:
Oops sorry, I think this also needs to be fixed in newapkbuild script as this was how I created it.